src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+
+/**
+ * Project Duke is a educational software project designed to take you
+ * through the steps of building a small software incrementally,
+ * while applying as many Java and SE techniques as possible along the way.
+ *
+ * @author AdrianOngJJ
+ * @version 0.1
+ * @since 22/1/2022
+ */
 public class Duke {
-    public static void main(String[] args) {
+    private static final String LINE_BREAK = "\n~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    private static final ArrayList<Task> masterList = new ArrayList<>();
+    /**
+     * Prints line break.
+     * @return void
+     */
+    private static final void printLineBreak() {
+        System.out.println(LINE_BREAK);
+    }
+
+    /**
+     * Prints Master List
+     * @param bw BufferedWriter from main to print out the Master List.
+     * @throws java.io.IOException If an I/O error occurs. Only takes in string.
+     */
+    private static final void printList(BufferedWriter bw) throws Exception {
+        for(int i = 0; i < masterList.size(); i++) {
+            Task curr = masterList.get(i);
+            bw.write((i + 1) + "." + curr.toString());
+            bw.newLine();
+        }
+        bw.flush();
+    }
+
+    private static final String getDateTime(String[] inputArr) {
+        return inputArr[1].split("/")[1].split(" ", 2)[1]; // split input from slash
+    }
+
+    private static final String getDescription(String[] inputArr) {
+        return inputArr[1].split("/")[0]; // split input from slash
+    }
+
+    public static void main(String[] args) throws Exception {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        printLineBreak();
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        String input; // to store raw input command
+        String[] inputArr; // to store split input command
+        boolean ifBye = false;
+        do {
+            printLineBreak();
+            System.out.println();
+            input = br.readLine();
+            inputArr = input.split(" ", 2); // split first word from body
+            printLineBreak();
+            switch (inputArr[0]) {
+            case "bye":
+                ifBye = true;
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+
+            case "list":
```

Comment:
Good job on following the guidelines with respect to the indentation levels of switch and case! There should be no need for line breaks between individual case statements, so maybe remove these as a final step?